albumentations/core/composition.py

```diff
@@ -201,11 +201,13 @@ def __call__(self, *args, force_apply: bool = False, **data) -> typing.Dict[str,
         for p in self.processors.values():
             p.preprocess(data)
 
+        data = self._make_targets_contiguous(data)  # some transforms require contiguous input
         for idx, t in enumerate(transforms):
             data = t(force_apply=force_apply, **data)
 
             if check_each_transform:
                 data = self._check_data_post_transform(data)
+            data = self._make_targets_contiguous(data)
```

Comment:
I think we do not need this check there because all transforms must return contiguous arrays. Better to write a test to check that each transform returns contiguous data.
Look at [test_image_only_augmentations_with_float_values](https://github.com/albumentations-team/albumentations/blob/master/tests/test_augmentations.py#L79) for an example

Reply:
Thank you for your comment.
OK, I think I got how the test works.
I will examine the violating transforms.